style/style.css

```diff
@@ -50,3 +50,71 @@ form button {
   display: flex;
   margin: 5px 0 0 auto;
 }
+
+form input {
+  margin-top: 10px;
+  font-size: 15px;
+}
+
+.nav {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  float: right;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Please add `hover` and `active` effect to the navigation items as shown in the wireframe.

![image](https://user-images.githubusercontent.com/63915024/182649468-f520e9da-27f3-4824-ad5d-5aba0182c5d1.png)

- ### Your design:

![image](https://user-images.githubusercontent.com/63915024/182650950-9d481ca7-6fcb-4db7-9c95-d6f328431f03.png)